src/Symfony/Component/Translation/TranslatorInterface.php

```diff
@@ -36,7 +36,7 @@ public function trans($id, array $parameters = array(), $domain = null, $locale
      * Translates the given choice message by choosing a translation according to a number.
      *
      * @param string      $id         The message id (may also be an object that can be cast to string)
-     * @param int         $number     The number to use to find the indice of the message
+     * @param float|int   $number     The number to use to find the indice of the message
```

Comment:
Yeah, you're right. Before closing this PR, IMO we should consider the fact to force that the Translator service can't works with decimal numbers. Allowing it to implicitly cast the number to an integer makes debugging harder than it should be.